### ISSUE TITLE
Expectile Loss?

### ISSUE THREAD

**Issue by haytham918** [pytorch/pytorch#118918]
### 🚀 The feature, motivation and pitch

I am doing a low-rank modeling with expectile analysis for my research, and I am wondering if there is an Expectile Loss Object(such as loss.ExpectileLoss()) that serves the purpose. I also tried to implement a custom loss function with expectile loss, but the results seem very strange(loss values are super small < 3).
